doc/source/reference/routines.array-creation.rst

```diff
@@ -47,7 +47,7 @@ From existing data
 Creating record arrays
 ----------------------
 
-.. note:: :mod:`numpy.core.records` is used to create record
+.. note:: ``numpy.core.records`` module is used to create record
```

Comment:
In this case, I think it'd be an improvement to avoid referring to `numpy.core.records` altogether. It's in `core` so it's not intended to be user-facing. All that's to say: I'd propose to delete the first sentence here altogether. Another alternative would be to replace `numpy.core.records` with it's public alias, `numpy.rec`.

Reply:
I would prefer to delete the first sentence since numpy.rec module don't have own section, only in codes.